editor/components/EditableField.tsx

```diff
@@ -108,9 +120,9 @@ export default function EditableField({
             <option value="false">No</option>
           </Form.Select>
         )}
-        {errors.value && (
+        {(errors.value || mutationError) && (
```

Comment:
if there is a validation error ie. you are typing in a string in an integer input, OR if you tried to save and got a mutation error, you will get an error message